engine/release-notes/index.md

```diff
@@ -22,14 +22,15 @@ for Docker Engine.
 
 # Version 19.03
 
-## 19.03.13 (2020-09-16)
+## 19.03.13
+2020-09-16
```

Comment:
Ah, thanks @usha-mandya, overlooked that

Reply:
No worries :)